client/src/style/squids/SquidShow.pcss

```diff
@@ -0,0 +1,35 @@
+.squid-show {
+  @apply container mx-auto mt-8 py-5 bg-white shadow-md border border-peach rounded;
+
+  & .squid-info {
+    @apply grid grid-cols-4 gap-4 mt-4;
+
+    & .squid__header {
+      @apply font-bold;
+    }
+
+    & .squid__text {
+      @apply font-normal;
+    }
+
+    & .squid__text-italic {
+      @apply font-normal italic whitespace-nowrap;
+    }
+
+    & li {
```

Comment:
In BEM, you'll want to only use elements as CSS selectors when absolutely absolutely necessary. Could you give these li's a descriptive class name instead?